__tests__/app.test.js

```diff
@@ -57,4 +57,41 @@ describe("/api test suite", () => {
         expect(body.endpoints).toEqual(endpoints);
             })
         })
-});
\ No newline at end of file
+});
+describe('GET /api/articles/:article_id', () => {
+    test('200: should respond with all articles that match the given article_id', () => {
+        return request(app)
+        .get('/api/articles/3')
+        .expect(200)
+        .then(({ body })=>{
+                expect(typeof body.article[0]).toBe("object"),
+                expect(body.article[0].article_id).toBe(3),
+                expect(body.article[0]).toHaveProperty(
```

Comment:
I would suggest changing this test to use .toMatchObject or object.Containing - this will make it more flexible for the future :)